frontend/src/metabase/visualizations/lib/renderer_utils.js

```diff
@@ -323,7 +323,7 @@ export function getXInterval({ settings, series }, xValues, warn) {
 
 export function xValueForWaterfallTotal({ settings, series }) {
   const xValues = getXValues({ settings, series });
-  const xInterval = getXInterval({ settings, series }, xValues);
+  const xInterval = getXInterval({ settings, series }, xValues, () => {});
```

Comment:
The last argument is for the `warn` function and IMO, it's not needed at all for computing this total's X value.

Reply:
Yeah that's fair. We would have already called `warn` if there's an issues.

Maybe needing to pass the no-op warn function is a sign we either split out the "warning" from `getXInterval` or pass the results of `getXInterval` into `xValueForWaterfallTotal`. Those both are messy and out of scope here though.